django_cloud_deploy/tool_requirements.py

```diff
@@ -287,12 +305,21 @@ def handle(cls, console: io.IO):
             elif answer in ['y', '']:
                 break
 
-        command = ['gcloud', '-q', 'components', 'install', 'cloud_sql_proxy']
-        if subprocess.call(command,
-                           stdout=subprocess.DEVNULL,
-                           stderr=subprocess.DEVNULL) != 0:
-            raise UnableToAutomaticallyInstallError(
-                cls.NAME, cls._AUTOMATIC_INSTALLATION_ERROR)
+        command = [gcloud_path, '-q', 'components', 'install', 'cloud_sql_proxy']
+        install_result = subprocess.run(command,
+                                        stdout=subprocess.DEVNULL,
+                                        stderr=subprocess.PIPE,
+                                        universal_newlines=True)
+        if install_result.returncode != 0:
+            if 'gcloud components update' in install_result.stderr:
+                raise UnableToAutomaticallyInstallError(
+                    cls.NAME, cls._OLD_GCLOUD_VERSION)
+            elif 'non-interactive mode' in install_result.stderr:
```

Comment:
Is stderr an array ? Or how is this non-interactive mode flag set?

Reply:
stderr is a string